src/dsp_tools/commands/xmlupload/models/ontology_diagnose_models.py

```diff
@@ -0,0 +1,102 @@
+# sourcery skip: use-fstring-for-concatenation
+
+from dataclasses import dataclass, field
+from pathlib import Path
+
+import pandas as pd
+
+from dsp_tools.models.exceptions import UserError
+
+
+@dataclass(frozen=True)
+class Ontology:
```

Comment:
we already have a class with this name in `src/dsp_tools/commands/project/models/ontology.py`, so we now have 2 classes with the same name in the codebase. However, the name `OntoInfo` is not taken yet, so perhaps you should rename it?